elasticsearchIndexing/es_manager.py

```diff
@@ -282,6 +282,19 @@ def get_sorted_module_revisions(self, index: ESIndices, name: str):
 
         return es_result['hits']['hits']
 
+    def get_maturity_levels_for_module(self, module_name: str):
```

Comment:
Doesn't this just duplicate the functionality of `get_sorted_module_revisions()`?

Reply:
not really, because get_sorted_module_revisions() gets info only about one module, so if I use it we are making many requests to elastic again like before

Reply:
I looked one more time, I think if I change ESIndex for old methods, I can find rfc without new methods